irc_bot/irc_bot.py

```diff
@@ -163,6 +163,7 @@ def __init__(self, config):
         self.invite_message = config.get('invite_message')
         self.nickserv_password = config.get('nickserv_password')
         self.use_ssl = config.get('use_ssl', False)
+        self.password = config.get('password', None)
```

Comment:
The python dict `get()` method returns `None` if the key does not exist. It's therefore not necessary to do `config.get('password', None)`. `config.get('password')` is sufficient.